packages/ui/src/components/Button/Button.tsx

```diff
@@ -56,6 +56,7 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
 					cssClass,
 				)}
 				ref={ref}
+				type="button"
```

Comment:
Could we make this accept a prop value for `type` and fallback to `"button"` as the default? In some cases we may want to use `type="submit"`.

Reply:
I think `buttonProps` includes a `type` prop which should be overriding this already